util-tracking/src/test/scala/sbt/util/TrackedSpec.scala

```diff
@@ -166,12 +157,82 @@ class TrackedSpec extends FlatSpec {
         val difference = System.currentTimeMillis - last
         assert(difference < 1000)
       }
+    }
+  }
+
+  "Difference.outputs" should "commit the cache will a snapshot of all input files after the execution" in {
+    withStore { store =>
+      val outCache = Difference.outputs(store, FileInfo.lastModified)
```

Comment:
Following the pattern, I think it would be better to call `Tracked.diffOutputs` here.